### ISSUE TITLE
[Opinion] Deleting obsolete services?

### ISSUE THREAD

**Issue by cibersheep** [loqui/im#1204]
Some of these services are obsolete / closed. It might be a good idea to do a check and decide if take them away.

* Nimbuzz: Used to be xmpp / SIP but now is ¿closed connection system?
Can't access with old user account

* Nokia Ove: Discontinued service [wikipedia](https://en.wikipedia.org/wiki/Nokia_Mail_and_Nokia_Chat)

* Microsoft Lync (No idea what's this). A [replaced by Skype](https://en.wikipedia.org/wiki/Skype_for_Business) service?

* Google Hangouts. Is it fixable? [wikipedia](https://en.wikipedia.org/wiki/Google_Hangouts)

* Facebook Chat. Old service? Is it possible to fix / upgrade?

**Comment by nfsprodriver** [comment]
If it's possible or wanted to reanimate them, this would be a good thing. Else we should hide them from the providers' selection list.

**Comment by Honk2** [comment]
Yes that is a good option. By this, the work to keep a provider working again does not start from scratch.

**Comment by bryanjhv** [comment]
Facebook was a good provider, but they replaced XMPP for MQTT. [bitlbee-facebook](https://github.com/bitlbee/bitlbee-facebook) got the new protocol working (even when there are no docs about it), and Loqui IM used to be the only app with this provider...
Maybe re-implementing the provider would be a good idea (and a lot of work to do, of course). :smiley:
